### PR TITLE
chore: add env folder in migrated generated yml

### PR DESCRIPTION
**Pull request by frankqianms**

### CODE REVIEW

File: packages/fx-core/templates/core/v3Migration/csharp.app.yml [hunk 1]
```diff
@@ -97,4 +99,4 @@ configureApp:
       appPackagePath: ./build/appPackage/appPackage.$\{{TEAMSFX_ENV}}.zip # Relative path to this file. This is the path for built zip file.
     # Output: following environment variable will be persisted in current environment's .env file.
     # TEAMS_APP_ID: the id of Teams app
-projectId: {{placeholderMappings.[projectId]}}
\ No newline at end of file
+projectId: {{projectId}}
```
Comment (blackchoey) [inline packages/fx-core/templates/core/v3Migration/csharp.app.yml:102]: can we put the projectId to the beginning of the yml file (together with environmentFolderPath)?

Comment (frankqianms) [reply packages/fx-core/templates/core/v3Migration/csharp.app.yml:102]: sure. good idea

**Review by blackchoey — DISMISSED** [review]

File: packages/fx-core/src/core/middleware/utils/appYmlGenerator.ts [hunk 2]
```diff
@@ -50,6 +52,8 @@ export class AppYmlGenerator extends BaseAppYmlGenerator {
       isWebAppBot: false,
       isTypescript: false,
       defaultFunctionName: undefined,
+      defaultEnvironmentFolder: undefined,
```
Comment (xzf0587) [inline packages/fx-core/src/core/middleware/utils/appYmlGenerator.ts:55]: In generator, the key name environmentFolder is better than defaultEnvironmentFolder.
Actually, we will refine it to use another folder when there is existing user's env folder in project

Comment (frankqianms) [reply packages/fx-core/src/core/middleware/utils/appYmlGenerator.ts]: changed

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**